versioned_docs/version-v4.0.0/node/backup-restore-update.md

```
@@ -0,0 +1,464 @@
+   
+    ```md
+    COMPOSE_PROFILES=hoprd docker compose down
+    ```
+
+3. **Update compose folder and complete configuration using previous version**
+
+    The following command assumes your HOPRd node folder is named `compose`. It downloads the latest compose folder, merges its files with your current `compose` folder, and updates the new configuration file with required fields from the previous configuration file.
+
+    Exit the `compose` folder and execute the following command:
+
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)